docs/cloud/tcld/account/metrics/accepted-client-ca/index.md

```diff
@@ -0,0 +1,23 @@
+---
+id: index
+title: tcld account metrics accepted-client-ca
+description: How to managed the client CA certificates for the metrics endpoint of a Temporal Cloud account using tcld.
+tags:
+  - reference
+  - tcld
+---
+
+The `tcld account metrics accepted-client-ca` commands manage the client CA certificates for the metrics endpoint of the Temporal Cloud account that is currently logged in.
+
+:::info
+
+The certificates for the metrics endpoint must chain up to the CA certificate used for the account. For more information, see [Requirements for CA certificates](/cloud/how-to-manage-certificates-in-temporal-cloud#requirements-for-ca-certificates).
```

Comment:
"The client certificates used when connecting to the metrics endpoint must chain up to"